package.json

```diff
@@ -79,6 +80,7 @@
   },
   "peerDependencies": {
     "dayjs": "^1.8.18",
+    "date-fns": "^2.15.0",
```

Comment:
I think we could ignore `peerDependencies` https://github.com/ant-design/ant-design/issues/23435

Reply:
OK